lib/npg/model/instrument.pm

```diff
@@ -380,6 +382,42 @@ sub latest_annotation {
   return $self->{latest_annotation};
 }
 
+sub recent_staging_volumes {
+  my $self = shift;
+
+  my $run_status = q[run in progress];
+  my $query = q[SELECT folder_path_glob FROM instrument
+                JOIN run USING(id_instrument)
+                JOIN run_status USING(id_run)
+                JOIN run_status_dict USING(id_run_status_dict)
+                WHERE folder_path_glob IS NOT NULL
+                AND LENGTH(folder_path_glob) != 0
+                AND id_instrument=? AND description=?
+                ORDER BY date DESC];
+  my $dbh = $self->util->dbh();
+  # Four rather than two latest runs since NovaSeq(X) instruments have
+  # two sides, which write to the same staging server.
+  my $rows = $dbh->selectall_arrayref($query, {RaiseError => 1, MaxRows => 4},
+    $self->id_instrument(), $run_status);
+  my @globs = uniq map { $_->[0] } @{$rows};
+  # We do not need more that two unique globs.
+  while (scalar @globs > 2) {
+    pop @globs
+  }
```

Comment:
```suggestion
                ORDER BY date DESC
                LIMIT 4 ];
  my $dbh = $self->util->dbh();
  # Four rather than two latest runs since NovaSeq(X) instruments have
  # two sides, which write to the same staging server.
  my $rows = $dbh->selectall_arrayref($query, {RaiseError => 1, MaxRows => 4},
    $self->id_instrument(), $run_status);
  my @globs = uniq map { $_->[0] } @{$rows};
  # We do not need more that two unique globs.
  while (scalar @globs > 2) {
    pop @globs
  }
```

Reply:
If this is to limit to 4 rows, then it is aleady done by `MaxRows => 4`